packages/client-sdk-nodejs/package-lock.json

```diff
@@ -12534,7 +12534,9 @@
       "link": true
     },
     "node_modules/@gomomento/generated-types": {
-      "version": "0.57.0",
+      "version": "0.59.0",
+      "resolved": "https://momento-prod-401011790710.d.codeartifact.us-west-2.amazonaws.com/npm/npm-upstream/@gomomento/generated-types/-/generated-types-0.59.0.tgz",
```

Comment:
this is why you are getting an npm error. These are dependencies installed via our internal/private npm registries. We should point all deps to the public npm registry. To fix this, you can remove this file, remove your `~/.npmrc` file, and `npm i` to reinstall all deps using the public registry